main.go

```diff
@@ -21,8 +23,7 @@ import (
 )
 
 func main() {
-	log.Println("Starting server on port " + os.Getenv("PORT") + "...")
-
+	log.Println("Main.go: Starting server on port " + os.Getenv("PORT") + "...")
```

Comment:
If we want file names and line numbers, we can wrap the log package and determine those at runtime.